libs/articles/feature-articles-list/src/lib/article-list.component.ts

```diff
@@ -4,13 +4,14 @@ import { Router } from '@angular/router';
 import { Store } from '@ngrx/store';
 import { articleListActions, articleListQuery, articlesActions } from '@realworld/articles/data-access';
 import { ArticleListItemComponent } from './article-list-item/article-list-item.component';
+import { PagerComponent } from '@realworld/ui/components/src';
```

Comment:
nitpick: It's not your fault , but could you remove the `/src`?

Reply:
Okay it's removed.